### PR TITLE
feat(oas): declare x-codegen on Admin routes - S to V

### PR DESCRIPTION
**Pull request by patrick-medusajs**
### What

Declare `x-codegen` in OAS for Admin routes - S to V.

### Why

We are introducing a new `x-codegen` OpenApi extension, also known as _vendor extension_, in order to help with passing information down to code generators.

In our case, we wish to declare the `method` name that we would expect to call on a client. This mimics our current JS client package. 
E.g. `medusaClient.product.list` where `product` is the tag of the route and `list` is the x-codegen.method value.

We are also defining the name of a potential typed object for query parameters. OAS 3.0 does not allow to bundle query parameters under a single definition but it is not uncommon to see API clients handle all query parameters as a single typed object, like our JS client package. With x-codegen.queryParams, a code generator can create a named and typed object to bundle all query parameters for a given route. 
E.g. `medusaClient.customer.retrieve(id: string, queryParams: AdminGetCustomerParams)` 

### How

Declare `x-codegen` as an object with fields `method` and `queryParams` on all paths.

Match method and queryParams values with equivalent ones from our current JS client package.

### Test

* Ran OAS validator.
* Ran docs build script.

Expect no visible changes to the documentation.

### CODE REVIEW

File: packages/medusa/src/api/routes/admin/shipping-options/update-shipping-option.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostShippingOptionsOptionReq"
+ * x-codegen:
+ *   method: update
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/shipping-options/update-shipping-option.ts:32]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/shipping-options.ts#L35-L39

File: packages/medusa/src/api/routes/admin/shipping-profiles/create-shipping-profile.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostShippingProfilesReq"
+ * x-codegen:
+ *   method: create
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/shipping-profiles/create-shipping-profile.ts:18]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/shipping-profiles.ts#L12-L15

File: packages/medusa/src/api/routes/admin/shipping-profiles/delete-shipping-profile.ts [hunk 3]
```diff
@@ -9,6 +9,8 @@ import { ShippingProfileService } from "../../../../services"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the Shipping Profile.
+ * x-codegen:
+ *   method: delete
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/shipping-profiles/delete-shipping-profile.ts:13]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/shipping-profiles.ts#L29-L32

File: packages/medusa/src/api/routes/admin/shipping-profiles/get-shipping-profile.ts [hunk 4]
```diff
@@ -13,6 +13,8 @@ import { ShippingProfileService } from "../../../../services"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the Shipping Profile.
+ * x-codegen:
+ *   method: retrieve
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/shipping-profiles/get-shipping-profile.ts:17]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/shipping-profiles.ts#L37-L40

File: packages/medusa/src/api/routes/admin/shipping-profiles/list-shipping-profiles.ts [hunk 5]
```diff
@@ -6,6 +6,8 @@ import { ShippingProfileService } from "../../../../services"
  * summary: "List Shipping Profiles"
  * description: "Retrieves a list of Shipping Profile."
  * x-authenticated: true
+ * x-codegen:
+ *   method: list
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/shipping-profiles/list-shipping-profiles.ts:10]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/shipping-profiles.ts#L45-L47

File: packages/medusa/src/api/routes/admin/shipping-profiles/update-shipping-profile.ts [hunk 6]
```diff
@@ -16,6 +16,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostShippingProfilesProfileReq"
+ * x-codegen:
+ *   method: update
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/shipping-profiles/update-shipping-profile.ts:20]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/shipping-profiles.ts#L20-L24

File: packages/medusa/src/api/routes/admin/stock-locations/create-stock-location.ts [hunk 7]
```diff
@@ -19,6 +19,8 @@ import { FindParams } from "../../../../types/common"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostStockLocationsReq"
+ * x-codegen:
+ *   method: create
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/stock-locations/create-stock-location.ts:23]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/stock-locations.ts#L21-L24

File: packages/medusa/src/api/routes/admin/stock-locations/get-stock-location.ts [hunk 8]
```diff
@@ -12,6 +12,9 @@ import { FindParams } from "../../../../types/common"
  *   - (path) id=* {string} The ID of the Stock Location.
  *   - (query) expand {string} Comma separated list of relations to include in the results.
  *   - (query) fields {string} Comma separated list of fields to include in the results.
+ * x-codegen:
+ *   method: retrieve
+ *   queryParams: AdminGetStockLocationsLocationParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/stock-locations/get-stock-location.ts:17]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/stock-locations.ts#L36-L39

File: packages/medusa/src/api/routes/admin/stock-locations/list-stock-locations.ts [hunk 9]
```diff
@@ -85,6 +85,9 @@ import { Request, Response } from "express"
  *   - (query) limit=20 {integer} Limit the number of stock locations returned.
  *   - (query) expand {string} (Comma separated) Which fields should be expanded in each stock location of the result.
  *   - (query) fields {string} (Comma separated) Which fields should be included in each stock location of the result.
+ * x-codegen:
+ *   method: list
+ *   queryParams: AdminGetStockLocationsParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/stock-locations/list-stock-locations.ts:90]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/stock-locations.ts#L81-L84

File: packages/medusa/src/api/routes/admin/stock-locations/update-stock-location.ts [hunk 10]
```diff
@@ -20,6 +20,8 @@ import { FindParams } from "../../../../types/common"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostStockLocationsLocationReq"
+ * x-codegen:
+ *   method: update
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/stock-locations/update-stock-location.ts:24]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/stock-locations.ts#L51-L55

File: packages/medusa/src/api/routes/admin/store/add-currency.ts [hunk 11]
```diff
@@ -17,6 +17,8 @@ import { EntityManager } from "typeorm"
  *       externalDocs:
  *         url: https://en.wikipedia.org/wiki/ISO_4217#Active_codes
  *         description: See a list of codes.
+ * x-codegen:
+ *   method: addCurrency
  * x-codeSamples:
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/store/add-currency.ts:22]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L31-L34

File: packages/medusa/src/api/routes/admin/store/get-store.ts [hunk 12]
```diff
@@ -15,6 +15,8 @@ import { ModulesHelper } from "../../../../utils/module-helper"
  * summary: "Get Store details"
  * description: "Retrieves the Store details"
  * x-authenticated: true
+ * x-codegen:
+ *   method: retrieve
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/store/get-store.ts:19]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L57-L59

File: packages/medusa/src/api/routes/admin/store/list-payment-providers.ts [hunk 13]
```diff
@@ -6,6 +6,8 @@ import { PaymentProviderService } from "../../../../services"
  * summary: "List Payment Providers"
  * description: "Retrieves the configured Payment Providers"
  * x-authenticated: true
+ * x-codegen:
+ *   method: listPaymentProviders
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/store/list-payment-providers.ts:10]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L68-L70

File: packages/medusa/src/api/routes/admin/store/list-tax-providers.ts [hunk 14]
```diff
@@ -6,6 +6,8 @@ import { TaxProviderService } from "../../../../services"
  * summary: "List Tax Providers"
  * description: "Retrieves the configured Tax Providers"
  * x-authenticated: true
+ * x-codegen:
+ *   method: listTaxProviders
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/store/list-tax-providers.ts:10]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L79-L81

File: packages/medusa/src/api/routes/admin/store/remove-currency.ts [hunk 15]
```diff
@@ -17,6 +17,8 @@ import { EntityManager } from "typeorm"
  *       externalDocs:
  *         url: https://en.wikipedia.org/wiki/ISO_4217#Active_codes
  *         description: See a list of codes.
+ * x-codegen:
+ *   method: deleteCurrency
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/store/remove-currency.ts:21]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L45-L48

File: packages/medusa/src/api/routes/admin/store/update-store.ts [hunk 16]
```diff
@@ -15,6 +15,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostStoreReq"
+ * x-codegen:
+ *   method: update
  * x-codeSamples:
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/store/update-store.ts:20]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L17-L20

File: packages/medusa/src/api/routes/admin/swaps/get-swap.ts [hunk 17]
```diff
@@ -10,6 +10,8 @@ import { SwapService } from "../../../../services"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the Swap.
+ * x-codegen:
+ *   method: retrieve
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/swaps/get-swap.ts:14]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/store.ts#L57-L59

File: packages/medusa/src/api/routes/admin/swaps/list-swaps.ts [hunk 18]
```diff
@@ -15,6 +15,9 @@ import { Swap } from "../../../../models"
  *   - (query) limit=50 {number} The upper limit for the amount of responses returned.
  *   - (query) offset=0 {number} The offset of the list returned.
  * x-authenticated: true
+ * x-codegen:
+ *   method: list
+ *   queryParams: AdminGetSwapsParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/swaps/list-swaps.ts:20]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/swaps.ts#L19-L22

File: packages/medusa/src/api/routes/admin/tax-rates/add-to-product-types.ts [hunk 19]
```diff
@@ -37,6 +37,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesTaxRateProductTypesReq"
+ * x-codegen:
+ *   method: addProductTypes
+ *   queryParams: AdminPostTaxRatesTaxRateProductTypesParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/add-to-product-types.ts:42]: Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L100-L105

File: packages/medusa/src/api/routes/admin/tax-rates/add-to-products.ts [hunk 20]
```diff
@@ -37,6 +37,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesTaxRateProductsReq"
+ * x-codegen:
+ *   method: addProducts
+ *   queryParams: AdminPostTaxRatesTaxRateProductsParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/add-to-products.ts:42]: Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L84-L89

File: packages/medusa/src/api/routes/admin/tax-rates/add-to-shipping-options.ts [hunk 21]
```diff
@@ -37,6 +37,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesTaxRateShippingOptionsReq"
+ * x-codegen:
+ *   method: addShippingOptions
+ *   queryParams: AdminPostTaxRatesTaxRateShippingOptionsParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/add-to-shipping-options.ts:42]: Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L116-L121

File: packages/medusa/src/api/routes/admin/tax-rates/create-tax-rate.ts [hunk 22]
```diff
@@ -39,6 +39,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesReq"
+ * x-codegen:
+ *   method: create
+ *   queryParams: AdminPostTaxRatesParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/create-tax-rate.ts:44]: Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L53-L57

File: packages/medusa/src/api/routes/admin/tax-rates/delete-tax-rate.ts [hunk 23]
```diff
@@ -9,6 +9,8 @@ import { TaxRateService } from "../../../../services"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the Shipping Option.
+ * x-codegen:
+ *   method: delete
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/delete-tax-rate.ts:13]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L180-L183

File: packages/medusa/src/api/routes/admin/tax-rates/list-tax-rates.ts [hunk 24]
```diff
@@ -71,6 +71,9 @@ import { validator } from "../../../../utils/validator"
  *       type: array
  *       items:
  *         type: string
+ * x-codegen:
+ *   method: list
+ *   queryParams: AdminGetTaxRatesParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/list-tax-rates.ts:76]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L39-L42

File: packages/medusa/src/api/routes/admin/tax-rates/remove-from-shipping-options.ts [hunk 25]
```diff
@@ -37,6 +37,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminDeleteTaxRatesTaxRateShippingOptionsReq"
+ * x-codegen:
+ *   method: removeShippingOptions
+ *   queryParams: AdminDeleteTaxRatesTaxRateShippingOptionsParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/remove-from-shipping-options.ts:42]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L164-L169

File: packages/medusa/src/api/routes/admin/tax-rates/update-tax-rate.ts [hunk 26]
```diff
@@ -40,6 +40,9 @@ import { isDefined } from "medusa-core-utils"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesTaxRateReq"
+ * x-codegen:
+ *   method: update
+ *   queryParams: AdminPostTaxRatesTaxRateParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/tax-rates/update-tax-rate.ts:45]: Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L68-L73

File: packages/medusa/src/api/routes/admin/users/create-user.ts [hunk 27]
```diff
@@ -17,6 +17,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminCreateUserRequest"
+ * x-codegen:
+ *   method: create
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/create-user.ts:21]: The current JS Client creates an alias to properly type the `role` property from TypeORM. 
The OAS abstracts the problem away by declaring `role?: enum: [admin, member, developer]`.
The codegen should then be able to output the property as `role?: 'admin' | 'member' | 'developer'`.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L64-L67

File: packages/medusa/src/api/routes/admin/users/delete-user.ts [hunk 28]
```diff
@@ -9,6 +9,8 @@ import UserService from "../../../../services/user"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the User.
+ * x-codegen:
+ *   method: delete
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/delete-user.ts:13]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L94-L97

File: packages/medusa/src/api/routes/admin/users/get-user.ts [hunk 29]
```diff
@@ -8,6 +8,8 @@ import UserService from "../../../../services/user"
  * x-authenticated: true
  * parameters:
  *   - (path) id=* {string} The ID of the User.
+ * x-codegen:
+ *   method: retrieve
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/get-user.ts:12]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L50-L53

File: packages/medusa/src/api/routes/admin/users/list-users.ts [hunk 30]
```diff
@@ -6,6 +6,8 @@ import UserService from "../../../../services/user"
  * summary: "List Users"
  * description: "Retrieves all users."
  * x-authenticated: true
+ * x-codegen:
+ *   method: list
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/list-users.ts:10]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L106-L108

File: packages/medusa/src/api/routes/admin/users/reset-password-token.ts [hunk 31]
```diff
@@ -14,6 +14,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminResetPasswordTokenRequest"
+ * x-codegen:
+ *   method: sendResetPasswordToken
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/reset-password-token.ts:18]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L22-L25

File: packages/medusa/src/api/routes/admin/users/reset-password.ts [hunk 32]
```diff
@@ -19,6 +19,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminResetPasswordRequest"
+ * x-codegen:
+ *   method: resetPassword
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/reset-password.ts:23]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L36-L39

File: packages/medusa/src/api/routes/admin/users/update-user.ts [hunk 33]
```diff
@@ -18,6 +18,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminUpdateUserRequest"
+ * x-codegen:
+ *   method: update
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/users/update-user.ts:22]: Client creates Request alias to type `role`. Not a problem for OAS codegen.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L79-L83

File: packages/medusa/src/api/routes/admin/variants/list-variants.ts [hunk 34]
```diff
@@ -64,6 +64,9 @@ import { IsType } from "../../../../utils/validators/is-type"
  *             gte:
  *               type: number
  *               description: filter by inventory quantity greater than or equal to this number
+ * x-codegen:
+ *   method: list
+ *   queryParams: AdminGetVariantsParams
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/variants/list-variants.ts:69]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/variants.ts#L17-L20

File: packages/medusa/src/api/routes/admin/variants/get-inventory.ts [hunk 35]
```diff
@@ -20,6 +20,8 @@ import { joinLevels } from "../inventory-items/utils/join-levels"
  * x-authenticated: true
  * parameters:
  *   - (path) id {string} The Product Variant id to get inventory for.
+ * x-codegen:
+ *   method: getInventory
```
Comment (patrick-medusajs) [inline packages/medusa/src/api/routes/admin/variants/get-inventory.ts:24]: https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/variants.ts#L37-L40

**Review by patrick-medusajs — COMMENTED** [review]
self-review: highlighting noteworthy code changes

**Review by olivermrbl — APPROVED** [review]
LGTM! 🚀

**Review by shahednasser — APPROVED** [review]
LGTM!

**Pull request merged**